examples/js/loaders/EXRLoader.js

```diff
@@ -76,11 +76,19 @@
 THREE.EXRLoader = function ( manager ) {
 
 	this.manager = ( manager !== undefined ) ? manager : THREE.DefaultLoadingManager;
+	this.type = THREE.FloatType;
 
 };
 
 THREE.EXRLoader.prototype = Object.create( THREE.DataTextureLoader.prototype );
 
+THREE.EXRLoader.prototype.setType = function ( value ) {
```

Comment:
You might want to add the new method `setType()` and the property `type` to the TS declaration file.